operator/controllers/kyma_controller.go

```diff
@@ -463,6 +467,13 @@ func (r *KymaReconciler) SetupWithManager(mgr ctrl.Manager, options controller.O
 				builder.WithPredicates(predicate.ResourceVersionChangedPredicate{}))
 	}
 
+	//register listener component
+	runnableListener, eventChannel := listener.RegisterListenerComponent(listenerAddr, strings.ToLower(v1alpha1.KymaKind))
+	//watch event channel
+	controllerBuilder.Watches(eventChannel, &handler.EnqueueRequestForObject{})
```

Comment:
still using enqueuerequestforobject instead of the custom change handler

Reply:
I think the handler should not be in the responsibility of the listener implementation. See README file in this [PR](https://github.com/khlifi411/kyma-listener/pull/2). I just provided this as a sample implementation in the PR. If you have other logic for the handler, I will remove this change, and you should provide your handler logic when invoking `Watches()`

Reply:
As discussed in our meeting, the custom handler logic will be included in the next iteration 👍